make/modules/java.desktop/lib/Awt2dLibraries.gmk

```diff
@@ -245,6 +245,8 @@ ifeq ($(call isTargetOs, windows macosx), false)
         DISABLED_WARNINGS_gcc_gtk3_interface.c := parentheses type-limits unused-function, \
         DISABLED_WARNINGS_gcc_OGLBufImgOps.c := format-nonliteral, \
         DISABLED_WARNINGS_gcc_OGLPaints.c := format-nonliteral, \
+        DISABLED_WARNINGS_gcc_screencast_pipewire.c := undef, \
```

Comment:
At least ignoring it just for the specific compilation units is not too bad.

I'll approve but I'd like to see a comment here as to why its needed - ie just for gcc6 so people have an idea when we might be able to remove it and know not to remove it because it doesn't matter for the current compiler version.

I've still asked upstream to consider applying your first version so that we'd not need this although perhaps it won't matter for long .. once no one cares about gcc6 because we need to move on for other reasons.

Reply:
There is no good way to put the comment straight into this block, AFAICS. Can we settle on repository history to tell future investigations why this was added?

Reply:
How about using a variable with a useful name? e.g.:

`DISABLED_FOR_NOT_COMPLETE_COMPILER_CHECK := undef`
...

```
        DISABLED_WARNINGS_gcc_OGLPaints.c := format-nonliteral, \
        DISABLED_WARNINGS_gcc_screencast_pipewire.c := $(DISABLED_FOR_NOT_COMPLETE_COMPILER_CHECK), \
        DISABLED_WARNINGS_gcc_screencast_portal.c := $(DISABLED_FOR_NOT_COMPLETE_COMPILER_CHECK), \
        DISABLED_WARNINGS_gcc_sun_awt_X11_GtkFileDialogPeer.c := parentheses, \
```

Reply:
Let's not over-complicate this? Most (all?) warning exclusions I saw were are added without checking for toolchain version. Limiting blast radius to just a toolchain and compilation unit seems good already.